test-suite/tests/error-p005-context.jsonld

```diff
@@ -0,0 +1,5 @@
+{
+  "@context": {
+    "@version": 1.1
```

Comment:
@lanthaler suggested a number, as using a string would be silently ignored by most 1.0 processors; the point is to use something that would cause a 1.0 processor to halt, thus the number.